ynr/apps/moderation_queue/review_required_helper.py

```diff
@@ -216,6 +216,26 @@ def needs_review(self):
             return self.Status.UNDECIDED
 
 
+class RevertedEditByAnotherUser(BaseReviewRequiredDecider):
+    def review_description_text(self):
+        return "Edit reverted by another user"
+
+    def needs_review(self):
+        if self.logged_action.user and self.logged_action.person:
+            update_qs = self.logged_action.person.loggedaction_set.filter(
+                action_type="person-update"
+            )
+
+            revert_qs = self.logged_action.person.loggedaction_set.filter(
+                action_type="person-revert"
+            )
+
+            if update_qs.exists() and revert_qs.exists():
+                if update_qs[0].user != revert_qs[0].user:
+                    return self.Status.NEEDS_REVIEW
+            return self.Status.UNDECIDED
```

Comment:
I might be wrong, but if the purpose of this work is to _always_ flag a revert of an edit, then I think this can be greatly simplified to just checking if the logged action is a `person-revert` or not can't it? E.g.:
```
if self.logged_action.action_type == ActionType.PERSON_REVERT:
    return self.Status.NEEDS_REVIEW
return self.Status.UNDECIDED
```

Reply:
I had understood that it was only to be flagged if the user reverting the edit was different than the person who made the edit.

Reply:
I think I figured it out; there was a mismatch with my assert.